client/luigi-client.js

```diff
@@ -1,307 +1,307 @@
+(function (root, factory) { // UMD wrapper
+  if (typeof define === 'function' && define.amd) {
+    define([], factory);
```

Comment:
From AMD documentation:
```
define(id?, dependencies?, factory);
```
Since dependencies are not required, we don't need this empty array of dependencies.